assets/javascripts/lib/discourse-markdown/bbcode-plugin.js

```diff
@@ -1,35 +1,23 @@
-// import "discourse/plugins/BBCode/bbcode-parser.min";
-
-// const bbcodeParser = window.bbcodeParser;
-
 /**
  * Processes inputted BBCode string using custom configured 3rd party library (see /bbcode-src)
  * @param {string} raw content to preprocess into HTML
  * @returns processed HTML string to pass into markdown-it
  */
 function preprocessor(raw) {
-  console.warn(...Object.getOwnPropertyNames(globalThis));
-  try {
```

Comment:
Couldn't quite figure out what this was trying to do. I've removed it as it currently outputs a lot into the console.

Reply:
Ahh, that was from me trying to figure out where and when was the plugin getting initialized. Basically needed a snapshot of the global namespace instead of a reference. All good to clear